### ISSUE TITLE
Add pcp metrics using `python3-pcp`

### ISSUE THREAD

**Issue by allisonkarlitskaya** [cockpit-project/cockpit#19036]
So this is a thing:

https://github.com/performancecopilot/pcp/blob/main/src/python/pcp/pmapi.py

And it's available as the `python3-pcp` package, at least on Fedora and Debian.

And even if it wasn't available, the fact that it's based on ctypes means that we could theoretically just dist it as part of the beipack.

We knew that we'd need to get the user to install PCP anyway in order to enable the metrics history.  If they're using the python bridge then we should just make sure to install `python3-pcp`.

Because this is Python we can do the import conditionally, meaning we don't even need a separate process for this anymore.

(We might consider a separate process still, though, if the rumours about the client library crashing are true...)

**Comment by jelly** [comment]
On Arch Linux it's available as part of the `pcp` package (not a split package):

```
>>> import pcp
>>> pcp.__file__
'/usr/lib/python3.11/site-packages/pcp/__init__.py'
```

**Comment by jelly** [comment]
Fixed in e9e1fcf58bb6fa8942e07000e80f336d860a90dd
